### PR TITLE
ehco: add livecheck

### PR DESCRIPTION
**Pull request by chenrui333**
ehco: add livecheck

---

```diff
- $ brew livecheck ehco
- ehco : 1.0.6 ==> 2020.06.11.833
+ $ brew livecheck ehco
+ ehco : 1.0.6 ==> 1.0.6
```

### CODE REVIEW

File: Formula/ehco.rb [hunk 1]
```diff
@@ -5,6 +5,11 @@ class Ehco < Formula
   sha256 "326c468c3790ad01031e52ccb4efcfa5e331d2198fdb13137749c67e8eaacf38"
   license "GPL-3.0-only"
 
+  livecheck do
+    url :stable
+    strategy :github_latest
```
Comment (carlocab) [inline Formula/ehco.rb:10]: We should tighten the regex used instead of using `:github_latest`.

**Review by carlocab — CHANGES_REQUESTED** [review]
We should tighten the regex used instead of using `:github_latest`.

File: Formula/ehco.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/ehco.rb:10]: yeah, I tried the option of using the regular regex, but it did not quite work out.

Comment (cho-m) [reply Formula/ehco.rb:10]: Assuming they don't add another dotted section, I guess you can restrict the number of sections:
```suggestion
    regex(/^v?(\d+(?:\.\d+){,2})$/i)
```

Comment (cho-m) [reply Formula/ehco.rb:10]: Or maybe:
```suggestion
    regex(/^v?(\d+(?:\.\d+){1,2})$/i)
```
since we prefer at least one dot?

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**